.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Thank you for contributing to vis.js!!**
+
+Please make sure to check the following requirements before creating a pull request:
+* [ ] All Pull-Requests must be against the [develop](https://github.com/almende/vis/tree/develop) branch. Pull-Request against `master` will be closed!
+* [ ] Make sure your changes are based on the latest version of the [develop](https://github.com/almende/vis/tree/develop) branch. (Use e.g. `git rebase origin develop` to update you feature-branch.)
```

Comment:
One dot at the end of the sentence i.e. `branch (use e.g. [..]).`

This appears in several places. Pls check.